instrumentation/k8s.io/client-go/splunkclient-go/option/option.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !(go1.1 || go1.2 || go1.3 || go1.4 || go1.5 || go1.6 || go1.7 || go1.8 || go1.9 || go1.10 || go1.11 || go1.12 || go1.13 || go1.14 || go1.15 || go1.16)
-// +build !go1.1,!go1.2,!go1.3,!go1.4,!go1.5,!go1.6,!go1.7,!go1.8,!go1.9,!go1.10,!go1.11,!go1.12,!go1.13,!go1.14,!go1.15,!go1.16
+//go:build go1.17
+// +build go1.17
```

Comment:
Maybe we can simply remove this lines? 

Take notice: https://github.com/signalfx/splunk-otel-go/blob/cdf7850435de19172273e62228b6ef1f49dc2580/instrumentation/k8s.io/client-go/splunkclient-go/go.mod#L3

It looks like a legacy code.

Reply:
Sounds reasonable, done